Core/Importers/KredoTransactionsImporter.cs

```diff
@@ -0,0 +1,73 @@
+﻿using iText.Kernel.Pdf;
+using iText.Kernel.Pdf.Canvas.Parser;
+using iText.Kernel.Pdf.Canvas.Parser.Listener;
+using iText.Layout;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+
+namespace Core.Importers
+{
+    class KredoTransactionsImporter : ITransactionsImporter
+    {
+        public IList<Transaction> Load(Stream file)
+        {
+            var saved = Thread.CurrentThread.CurrentCulture;
+
+            try
+            {
+                Thread.CurrentThread.CurrentCulture = CultureInfo.CreateSpecificCulture("en-GB");
+                Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+
+                var pdfDoc = new PdfDocument(new PdfReader(file));
+                var listener = new FilteredEventListener();
+                var extractionStrategy = listener.AttachEventListener(new SimpleTextExtractionStrategy());
+
+                var transactions = new List<Transaction>();
+                var pagesAmount = pdfDoc.GetNumberOfPages();
+
+                for (int i = 0; i < pagesAmount; i++)
+                {
+                    new PdfCanvasProcessor(listener).ProcessPageContent(pdfDoc.GetPage(i + 1));
+
+                    var actualText = extractionStrategy.GetResultantText().Split("\n").Skip(28).SkipLast(2).ToArray();
+                    var cardNumber = actualText[0];
+                    var category = "КредоБанк";
```

Comment:
Codacy found an issue: [Add the 'const' modifier to 'category'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5961168)